src/WrappedGovLst.sol

```diff
@@ -136,6 +136,9 @@ contract WrappedGovLst is ERC20Permit, Ownable {
   /// @return _wrappedAmount The quantity of wrapped tokens issued to the caller.
   /// @dev The caller must approve at least the amount of tokens to wrap on the lst contract before calling. Amount to
   /// wrap may not be zero.
+  /// @dev When wrapping a user may send at most 1 extra wei of `GovLst` tokens to the `WrappedGovLst` due
+  /// to an initial `transferFrom`. A second transfer of `GovLst` to the fixed alias address on the
+  /// `FixedGovLst` will use the initial wrapped amount rather than the converted amount ensuring that at most 1 extra wei is sent rather than 2.
```

Comment:
```suggestion
  /// will use the initial wrapped amount rather than the converted amount ensuring that at most 1 extra wei is sent rather than 2.
```